content/docs/guides/multi-target-exporter.md

```diff
@@ -0,0 +1,448 @@
+* You can set internal labels that are called `__param_<name>`. Those set URL parameter with the key `<name>` for the scrape request.
+* There is an internal label `__address__` which is set by the `targets` under `static_configs` and whose value is the hostname for the scrape request. By default it is later used to set the value for the label `instance`, which is attached to each metric and tells you were the metrics came from.
+
+Here is the config you will use to do that. Don’t worry if this is a bit much at once, we will go through it step by step:
+
+```yaml
+scrape_configs:
+
+- job_name: blackbox-exporter-metrics
+  scrape_interval: 5s
+  metrics_path: /metrics
+  static_configs:
+    - targets:
+      - localhost:9115   # for Windows and macOS replace with - host.docker.internal:9115
+
+- job_name: 'blackbox'
+  scrape_interval: 5s
+  metrics_path: /probe
+  params:
+    module: [http_2xx]
+  static_configs:
+      - targets:
+        - http://prometheus.io    # Target to probe with http.
+        - https://prometheus.io   # Target to probe with https.
+        - http://example.com:8080 # Target to probe with http on port 8080.
+  relabel_configs:
+    - source_labels: [__address__]
+      target_label: __param_target
+    - source_labels: [__param_target]
+      target_label: instance
+    - target_label: __address__
+      replacement: localhost:9115  # The blackbox exporter’s real hostname:port. For Windows and macOS replace with - host.docker.internal:9115
+```
+
+So what is new compared to the [last config](#prometheus-config)?
+
+ 1. `params` does not include `target` anymore.
+ 1. We add the actual targets under `static configs:` `targets`. We also use several because we can do that now.
+ 1. `relabel_configs` contains the new relabeling rules.
+ 1. We take the values from the label `__address__` (which contain the values from `targets`) and write them to a new label `__param_target` which will add a parameter `target` to the Prometheus scrape requests.
+ 1. We take the values from the label `__param_target` and create a label instance with the values.
+ 1. We write the value `localhost:9115` to the the label `__address__`. This will be used as the hostname and port for the Prometheus scrape requests.
+
+This way we can have the actual targets there, get them as `instance` label values while letting Prometheus make a request against the blackbox exporter.
+
+Often people combine these with a specific service discovery. Check out the [configuration documentation](/docs/prometheus/latest/configuration/configuration) for more information. Using them is no problem, as these write into the `__address__` label just like `targets` defined under `static_configs`.
+
+That is it. Restart the Prometheus docker container and look at your [metrics](http://localhost:9090/graph?g0.range_input=5m&g0.stacked=0&g0.expr=probe_http_duration_seconds&g0.tab=0).
```

Comment:
Maybe a closing sentence about what differences to look for, or how to extend the time range of the graph in case they took longer to do it?